ui/app/rif/pages/addNewMulticryptoAddress/addNewMulticryptoAddress.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronLeft } from '@fortawesome/free-solid-svg-icons'
+import { CustomButton } from '../../components'
+import rifActions from '../../actions'
+import { cryptos } from '../../constants'
+
+class AddNewMulticryptoAddressScreen extends Component {
+	state = {
+		networks: [],
+		address: '',
+		network: '',
+	}
+	constructor(props) {
+		super(props);
+		let networks = new Set()
+		let selectFirst = true
+		let selected = ''
+		Object.keys(cryptos).forEach(function(key) {
+			networks.add(key)
+			if(selectFirst){
+				selected = key
+				selectFirst = false
+			}
+		});
+		this.state = { 
+			networks: [...networks],
+			network: selected,
+		};
+	}
+
+	navigateTo (url) {
```

Comment:
This function is necessary? is even used? I can't see any reference, i think we could remove it

Reply:
Please check out, as it now is a modal